nss-cohort-37/c37-class-website
# Updated Technologies

## Description of Proposed Changes

Updated the technologies panel to display the relevant technologies we used.

## Steps to Test

Check to see if images & links are displaying in tech panel.

```sh
git fetch --all
git checkout jv-technologies

```


## Impacted Areas in Application

**REMOVED**
Angular
Grunt
Browserify
Materialize
SQLLite
Uglify

**ADDED**
React
JSON
Postman
DbDiagram
Figma
Cloudinary
